### PR TITLE
Fix deprecated set-output command with $GITHUB_OUTPUT (Software-5354)

### PR DESCRIPTION
**Pull request by jeff-takaki**
Replaced with updated syntax echo "KEY=VALUE >>$GITHUB_OUTPUT"

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -65,8 +65,7 @@ runs:
         echo "No image_name provided and ${{ inputs.context }} is not a directory"
         exit 1
       fi
-      echo "::set-output name=image::${IMAGE_NAME,,}"
-
+      echo "image=${IMAGE_NAME,,}" >>GITHUB_OUTPUT
```
Comment (brianhlin) [inline action.yml]:
```suggestion
      echo "image=${IMAGE_NAME,,}" >> GITHUB_OUTPUT
```

File: action.yml [hunk 2]
```diff
@@ -92,7 +91,7 @@ runs:
         fi
       done
       IFS=,
-      echo "::set-output name=taglist::${tag_list[*]}"
+      echo "taglist=${tag_list[*]}" >>$GITHUB_OUTPUT
```
Comment (brianhlin) [inline action.yml]:
```suggestion
      echo "taglist=${tag_list[*]}" >> $GITHUB_OUTPUT
```

File: action.yml [hunk 3]
```diff
@@ -121,4 +120,3 @@ runs:
         TIMESTAMP_TAG=${{ inputs.timestamp_tag }}
       tags: "${{ steps.generate-tag-list.outputs.taglist }}"
       cache-from: type=local,src=/tmp/.buildx-cache
-
```
Comment (brianhlin) [inline action.yml]: Extraneous whitespace change :(

**Review by brianhlin — APPROVED** [review]
Minor suggestions but otherwise LGTM

**Comment by jeff-takaki** [comment]
@brianhlin I think white space issue should be okay now.

**Review by brianhlin — APPROVED** [review]
LGTM!

**Pull request merged**